tests/test_argument_check.py

```diff
@@ -31,7 +31,7 @@ def create_settings(test_type, vcs_type):
         settings.TeamcityServer.user_id = "TeamCityUser"
         settings.TeamcityServer.passwd = "TeamCityPassword"
 
-    if vcs_type == "git" or vcs_type == "gerrit" or vcs_type == "github":
+    if vcs_type in ("git", "gerrit", "github",):
```

Comment:
In my opinion, lists look cleaner than tuples

Reply:
fixed